trulens_eval/examples/quickstart/langchain_retrieval_multi_agents.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>Because you're using deferred mode, you need to start the evaluator at some point. Can you also write some markdown on why the deferred mode is useful here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
yess - added on the top